l10n_it_fatturapa_export_zip/wizard/export_fatturapa.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+
+from odoo import models, api, fields, _
+import base64
+import io
+import zipfile
+
+
+class WizardAccountInvoiceExport(models.TransientModel):
+    _name = "wizard.fatturapa.export"
+
+    data = fields.Binary("File", readonly=True)
+    name = fields.Char('Filename', size=32)
+    mark_as_exported = fields.Boolean('Mark as exported', default=True)
+
+    @api.multi
+    def export_zip(self):
+        attachments = []
+        for att in self.env[self._context['active_model']].browse(
+                self._context['active_ids']):
+            attachments += [att]
+
+        fp = io.BytesIO()
+        zf = zipfile.ZipFile(fp, mode="w")
+
+        for att in attachments:
+            zf.writestr(att.datas_fname, base64.b64decode(att.datas))
+        zf.close()
+        fp.seek(0)
+        data = fp.read()
+        export_report_name = 'E-Invoices XML'
+        if self.name:
+            export_report_name = self.name
+        attach_vals = {
+            'name': export_report_name + '.zip',
+            'datas_fname': export_report_name + '.zip',
+            'datas': base64.encodestring(data),
+        }
+        att_id = self.env['ir.attachment'].create(attach_vals)
```

Comment:
L'allegato `att_id` lo allegherei all'esportazione e-fattura (tramite `res_model` e `res_id`). Altrimenti poi se chiudo la pagina dello ZIP, me lo perdo.

Reply:
l'allegato ad ogni e-fattura? non ho capito

Reply:
@sergiocorato giusto, no, non ad ogni fattura. Il problema che vorrei risolvere è: una volta esportato lo ZIP, in un secondo momento vorrei poter recuperare lo ZIP agevolmente.

Piuttosto allora aggiungere un m2o verso `ir.attachment` su `fatturapa.attachment.out` e `fatturapa.attachment.in`, dove salvare il riferimento allo ZIP?

A quel punto, volendo, `zip_exported` potrebbe diventare calcolato